scripts/directives/cypress-visit-mount-test-example.js

```diff
@@ -0,0 +1,41 @@
+const re = /( *)__VISIT_MOUNT_PLACEHOLDER__/g
```

Comment:
I feel like our standard indent has been 2 spaces. Is there a reason that you know of why this file is being formatted differently? I'm seeing a mix of tabs and 4-space indents.

Reply:
That _is_ strange. I'll look into the lint hooks.